src/textual/renderables/bar.py

```diff
@@ -95,6 +97,16 @@ def __rich_console__(
 
         yield output_bar
 
+    def _get_thickness_code(self) -> tuple(str, str, str):
```

Comment:
This method could do with a better name, please!
And this could also be made a property instead of a getter method.

```suggestion
    def _get_bar_characters(self) -> tuple(str, str, str):
```

```suggestion
    @property
    def _bar_characters(self) -> tuple(str, str, str):
```

Reply:
Both fixed.